oasis/models/rules.yml

```diff
@@ -0,0 +1,17 @@
+# support features: mean, std, var, sum, min, max
+# rules:
+# features["std"]
```

Comment:
Be consistent. Here we use `feature["std"]`, why later switch to `feature[std]`?